packages/cli/src/blueprints/.eslintrc.cjs

```diff
@@ -32,11 +29,19 @@
     // TypeScript files
     {
       files: ['**/*.{cts,ts}'],
-      extends: ['plugin:@typescript-eslint/recommended-type-checked'],
+      parser: '@typescript-eslint/parser',
```

Comment:
I updated the `eslint` configuration so that it may be clearer what is actually needed for `parser` and `extends` for (1) TypeScript files and (2) JavaScript files.